verl/trainer/main_ppo.py

```diff
@@ -67,6 +67,9 @@ def run_ppo(config) -> None:
         and config.trainer.get("profile_steps") is not None
         and len(config.trainer.get("profile_steps", [])) > 0
     ):
+        from verl.utils.import_utils import is_nvtx_available
+
+        assert is_nvtx_available(), "nvtx is not available in CUDA platform. Please 'pip3 install nvtx'"
```

Comment:
Thank you for saving me another day. I don't know why it didn't show any errors even though I didn't have `nvtx` installed (only `nvidia-nvtx-cu12`) . It generated the nsys report, but without NVTX and CUDA hardware information. After installing it, everything shows up correctly.